frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -219,30 +254,62 @@ interface ILandForm {
   handlePinChange: (pin: string) => void;
   /** whether or not this user has admin priviledges */
   isAdmin: boolean;
+  /** initial values used to populate this form */
+  initialValues: IFormParcel;
+  /** whether this form can be interacted with */
+  disabled?: boolean;
 }
 
 interface IParentLandForm extends ILandForm {
   /** signal the parent that the associated land process has been completed. */
   setLandComplete: (show: boolean) => void;
 }
 
+/**
+ * A wrapper around the landform that provides all expected props for the landform to be used in read-only mode.
+ * This sets all the fields to disabled and disables all actions triggered by interacting with the forms.
+ * It also disables form validation and submission.
+ * @component
+ */
+export const ViewOnlyLandForm: React.FC<Partial<IParentLandForm>> = (props: {
+  formikRef?: any;
+  initialValues?: IFormParcel;
+}) => {
+  return (
+    <LandForm
+      setMovingPinNameSpace={noop}
+      formikRef={props.formikRef}
+      handleGeocoderChanges={async (response: IGeocoderResponse) => {}}
+      handlePidChange={noop}
+      handlePinChange={noop}
+      isAdmin={false}
+      setLandComplete={noop}
+      initialValues={props.initialValues ?? ({} as any)}
+      disabled={true}
+    />
+  );
+};
+
 /**
  * A component used for submitting bare land.
  * This form will appear after selecting 'Add Bare Land' after navigating to Manage Property > Submit Property in PIMS
  * @component
  */
-
 const LandForm: React.FC<IParentLandForm> = (props: IParentLandForm) => {
   const keycloak = useKeycloakWrapper();
   const dispatch = useDispatch();
   const api = useApi();
-  let initialValues = {
+  let initialValueProps = {
     activeStep: 0,
     activeTab: 0,
-    data: getInitialValues(),
+    tabs: [{ activeStep: 0 }],
+    data: { ...getInitialValues(), ...props.initialValues },
   };
+  const [initialValues, setInitialValues] = useState<ISteppedFormValues<IFormParcel>>(
```

Comment:
need to set the initial props values here, so that this can be overridden by the formikpersist component in case the user wants to load a draft.